src/java.se/share/classes/module-info.java

```diff
@@ -35,6 +35,8 @@
  * </dd>
  * </dl>
  *
+ * @spec jdwp/jdwp-spec.html Java Debug Wire Protocol
+ * @spec jni/index.html Java Native Interface Specification
```

Comment:
One thing that that bothers me a bit here is that the JNI and JDWP specs will be listed as "External Specifications" in the generated javadoc. This heading is appropriate for RFCs and other standards that we reference but seems misleading for specifications that are part of Java SE. Has this come up already?